zerver/lib/actions.py

```diff
@@ -4960,6 +4960,37 @@ def notify_user_update_custom_profile_data(user_profile: UserProfile,
     event = dict(type="realm_user", op="update", person=payload)
     send_event(event, active_user_ids(user_profile.realm.id))
 
+def validate_user_custom_profile_data(realm_id: int,
```

Comment:
Let's put this in `zerver/lib/users.py`; that has a lot of similar validation code.